client/scss/styleguide/_styleguide.scss

```diff
@@ -1,122 +1,5 @@
-.styleguide__heading {
-  margin-bottom: 2rem;
-}
-
-.styleguide__logo-wrap {
-  margin-bottom: 4rem;
-  text-decoration: none;
-  font-weight: bold;
-  display: block;
-}
-
-.styleguide__logo {
-  width: 210px;
-  height: 45px;
-}
-
-.styleguide__subheading {
-  line-height: 1;
-  display: block;
-}
-
-.styleguide__main {
-  position: absolute;
-  left: 270px;
-  overflow-y: scroll;
-  top: 0;
-  bottom: 0;
-  right: 0;
-  background: color('white');
-  transition: left 200ms ease;
-  border-left: 1px solid color('smoke');
-  padding: 60px 15px 15px;
-
-  &.is-fullscreen {
-    left: 0;
-  }
-}
-
-.styleguide__top-nav {
-  position: absolute;
-  top: 15px;
-  left: 0;
-  height: 30px;
-  border-bottom: 1px solid color('smoke');
-  right: 0;
-}
-
-.styleguide__menu {
-  position: relative;
-  display: block;
-  width: 25px;
-  height: 25px;
-  margin-left: 15px;
-  cursor: pointer;
-
-  span {
-    display: block;
-    height: 4px;
-    margin-bottom: 3px;
-    background: color('light-grey');
-    transition: background 200ms ease;
-  }
-
-  &:hover,
-  &:focus {
-    span {
-      background: color('charcoal');
-    }
-  }
-}
-
-.styleguide__sidebar {
-  padding: 15px;
-  background: color('mist');
-  top: 0;
-  bottom: 0;
-  width: 270px;
-  position: absolute;
-  overflow-y: auto;
-}
-
-.styleguide__list {
-  @include plain-list();
-}
-
-.styleguide__item {
-  font-size: $font-xsmall;
-  color: color('granite');
-  text-transform: uppercase;
-  font-weight: 500;
-  margin: 0 0 0.4em;
-
-  .styleguide__list--small & {
-    text-transform: none;
-    font-weight: normal;
-    margin-left: 1rem;
-  }
-
-  &:last-child {
-    margin-bottom: 1.6em;
-  }
-}
-
-.styleguide__link {
-  display: block;
-  text-decoration: none;
-  padding-right: 1rem;
-  border-bottom: 0;
-  transition: color 200ms ease;
-
-  &:hover,
-  &:focus,
-  &.is-active {
-    color: color('blue');
-  }
-}
-
-.styleguide__standalone {
-  margin-bottom: 30px;
+.container {
```

Comment:
I thought things were meant to be shared - or is this in relation to Fractal's shell?